google/web-stories-wp
# UI Simplification: Selection Panel

## Summary
Simplifies the Selection panel (previously Size & Position)


## Relevant Technical Choices



## To-do

- [x] Fix grid area lint



## User-facing changes
| Before | After |
| ------ | ----- |
| <img width="306" alt="Screenshot 2021-10-26 at 17 27 41" src="https://user-images.githubusercontent.com/3294597/138899988-c9fa39d1-62de-4128-9b91-c10a3afb62fd.png"> | <img width="301" alt="Screenshot 2021-10-26 at 17 26 08" src="https://user-images.githubusercontent.com/3294597/138899636-3c487408-8940-43d1-95be-cb3bdc5b6306.png"> |
| <img width="302" alt="Screenshot 2021-10-26 at 17 27 50" src="https://user-images.githubusercontent.com/3294597/138900040-82c51683-61a7-4317-bbc1-bd4d3d1775ea.png"> | <img width="305" alt="Screenshot 2021-10-26 at 17 25 53" src="https://user-images.githubusercontent.com/3294597/138899730-1661334b-6dcf-47f1-97e7-4bd5cf4f0e3e.png"> |


## Testing Instructions






This PR can be tested by following these steps:

1. Verify that the Layer panel is not a separate panel anymore and is now under the new Selection panel (verify for each element type).
2. Verify that Opacity can be changed from the Opacity input (previously was in the Layer panel)
3. Verify that Corner radius panel is not a separate panel anymore and is now an input under the Selection panel.
4. Verify the unlocked and locked corner radius input style per [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor-Sprint-Feature-Level-Designs_?node-id=10250%3A436882) for the corner radius input(s)
5. Verify the corner radius works as expected still.

Note: the opacity input icon (the drop) looks a bit different than in Figma, that will be fixed in another PR.

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9460
